sktime/transformations/series/clear_sky.py

```diff
@@ -40,17 +41,24 @@ class ClearSky(BaseTransformer):
 
     Parameters
     ----------
-    quantile_prob : float, default=0.95
```

Comment:
I think there should be a space on both sides. Currently not testing but it's numpydoc style afaik